### DISCUSSION TITLE
Should catalog numbers include the institution code or collection code?

### DISCUSSION THREAD

**Discussion by zygoballus** [Symbiota/community-board#675] [Q&A] [answered]
My institution code is CMNH (for Carnegie Museum of Natural History) and my collection code is IZ (for invertebrate zoology). So far, I've been entering my catalog numbers as e.g. "CMNH-IZ-1", "CMNH-IZ-2", but I noticed that some other collections only include the collection code in their catalog numbers, or neither the institution code nor the collection code. And some include both like I do. Is there any best practice in this regard? Is it good to use a globally unique catalog number in the field or is it superfluous to include the institution code and collection code? The Darwin Core documentation just says "An identifier (preferably unique) for the record within the data set or collection."

**Comment by zygoballus** [comment]
Any opinions are welcome!

**Reply by ghost** [reply, marked as answer]
Hi @zygoballus,

My opinion is that it is best not to include the collection code in the catalogue number, thus keeping your data atomic (small, independent components with one function). If the three should be shown together, such as in an interface, then that interface is responsible for concatenating the institution code, collection code, and catalogue number together. The same thought process is why it is better to separate out first and last name, as opposed to having a single field, you have a lot more flexibility with it.

Additionally, separating out the institution code and collection code deprives the catalogue number of meaning. While this may sound bad, things that are meaningless don't tend to change very often, which is great for numbers used to identify things. If the name of the museum or collection changes (unlikely, but it does certainly happen), then there may be temptation down the line to come up with the new scheme, and renumber etc.

**Comment by fbungartz** [comment]
Makes no sense to change it now, because different collections use different catalog numbers (typically barcodes) some with others without collecting acronym. Enforcing everyone to follow one standard would mess with current practice.

For example, at ASU we use ASUL000000 for lichens, ASUL000000 for bryophytes, ASUC000000 for cyanobacteria, but ASU0000000 for vascular plants - barcode encoding format for vascular plants code 42, for all others code 128.
Reasons for this are somewhat historical. The vascular plant collection started using archival barcode stickers first wirh size less of an issue (code 32 is generally larger than 128). The other collections used barcodes without prefix on paper. When some of these became illegible (the quality of some prints fading), we switched to atchtival stickers but due to packet size needed a different format (code 128).

Long story short: archival barcodes are expensive and we cannot afford to switch again simply because some folks may prefer catalog numbers and collection acronyms to be databased separately.
